packages/web-components/fast-components/src/skeleton/skeleton.styles.ts

```diff
@@ -0,0 +1,60 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import {
+  neutralFillRestBehavior
+} from "../styles";
+
+export const SkeletonStyles = css`
+  ${display("block")} :host {
+    overflow: hidden;
+    width: 100%;
+    position: relative;
+    background-color: ${neutralFillRestBehavior.var};
+    --skeleton-animation-gradient-default: linear-gradient(270deg, ${neutralFillRestBehavior.var} 0%, #F3F2F1 51.13%, ${neutralFillRestBehavior.var} 100%);
+  }
+
+  :host(.rect) {
+    border-radius: calc(var(--corner-radius) * 1px);
+  }
+
+  :host(.circle) {
```

Comment:
@nicholasrice Should we use attribute selectors for this? `:host[shape=circle]` instead of making it class based? Will that help out React scenarios?

Reply:
I'm going to resolve this for now. We can update again later once a decision is made.